### PR TITLE
feat(detailsPanel): add detailsPanel in list-view

### PR DESCRIPTION
**Pull request by Birkbjo**
https://dhis2.atlassian.net/browse/DHIS2-14652

### CODE REVIEW

File: src/components/sectionList/SectionListWrapper.tsx [hunk 1]
```diff
@@ -76,33 +79,53 @@ export const SectionListWrapper = <Model extends IdentifiableObject>({
...
-                                    sectionModel={model}
-                                />
-                            )
-                        }}
-                    />
-                ))}
-                <SectionListPagination data={data} />
-            </SectionList>
+            <div className={css.listDetailsWrapper}>
+                <div className={css.listToolbarWrapper}>
```
Comment (Mohammer5) [inline src/components/sectionList/SectionListWrapper.tsx:83]: Just a nitpick: This div isn't necessary, you could assign it to its own area. I don't care too much

File: src/components/sectionList/SectionList.module.css [hunk 2]
```diff
@@ -6,10 +6,33 @@
...
     height: 48px;
     padding: var(--spacers-dp8);
-    margin-top: var(--spacers-dp8);
     gap: var(--spacers-dp8);
 }
 
+.listToolbarWrapper {
+    display: flex;
+    flex-direction: column;
+    grid-area: 'list';
```
Comment (Mohammer5) [inline src/components/sectionList/SectionList.module.css:15]:
```suggestion
    grid-area: list;
```

File: package.json [hunk 3]
```diff
@@ -48,5 +48,8 @@
         "use-query-params": "^2.2.1",
         "zod": "^3.22.2",
         "zustand": "^4.4.0"
+    },
+    "resolutions": {
+        "@dhis2/ui": "8.14.2-alpha.1"
```
Comment (Mohammer5) [inline package.json:53]: Is alpha up2date with ui@master? I suppose that in that case this is fine, otherwise it might cause issues.

**Review by Mohammer5 — COMMENTED** [review]
LGTM, just a couple minor things

File: src/components/sectionList/SectionListWrapper.tsx (hunk 1 above)
Comment (Birkbjo) [reply src/components/sectionList/SectionListWrapper.tsx:83]: Not sure which one you are referring to? Pretty sure both are required for the required functionality. If listToolbarWrapper was removed the toolbar-actions would be affected by the listDetails-grid (that is supposed to target the list and details). These also handle the responsive functionality of the list.

File: package.json (hunk 3 above)
Comment (Birkbjo) [reply package.json:53]: We may want to wait until including this? Maybe in a separate PR?

File: src/components/sectionList/SectionListWrapper.tsx (hunk 1 above)
Comment (Mohammer5) [reply src/components/sectionList/SectionListWrapper.tsx:83]: See here: #364

File: package.json (hunk 3 above)
Comment (Mohammer5) [reply package.json:53]: Yeah maybe you could extract the upgrade to ui@alpha and we handle adding the details panel separate from that (but we can keep the fixes for wrong prop usage here, would be a shame to have those)

File: src/components/sectionList/SectionListWrapper.tsx (hunk 1 above)
Comment (Birkbjo) [reply src/components/sectionList/SectionListWrapper.tsx:83]: Thanks, that makes sense! Didn't understand the "assign it to it's own area" as adding it to the grid. Will try it out!

**Review by Mohammer5 — APPROVED** [review]

**Pull request merged**